File: Geklaut/06_Spring_Wirtschaftstag_Janssen/src/main/java/janssen/app/models/Event.java
package janssen.app.models;

import lombok.Data;

import javax.persistence.*;

import java.time.LocalDate;

@Data
@Entity
public class Event {

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private int id;
  private String label;
  private LocalDate date;
  private double defaultPrice;

  @ManyToOne
  private Admin organiser;

}

File: KOTLIN/Gstohlen/Kotlin_Wirtschaftstag/app/src/main/java/at/eder/wirtschaftstagmobileapp/models/Event.kt
package at.scarc.androidApp.models

import java.time.LocalDate
import java.time.LocalDateTime

data class Event (
    var id: Long?,
    var label: String?,
    var date: String?,
    var organiser: User?
) {
    override fun toString(): String = "$id - $label: $date"
}
